.storybook/preview.js

```diff
@@ -25,7 +25,8 @@ function storySort(a, b) {
     prefix('labs-', '4'),
     prefix('basic', 'aa'),
     prefix('default', 'ab'),
-    prefix('visual-testing', 'zz')
+    prefix('visual-testing', 'zz'),
+    prefix('examples', 'zzz')
```

Comment:
This will put examples at the very bottom of our sidenav list in Storybook. I think that's fine for now. When we figure out how we want to present examples to people, we can find a more visible location for it.